modules/tinymce/src/plugins/table/test/ts/browser/TableInsertionTest.ts

```diff
@@ -0,0 +1,39 @@
+import { context, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/table/Plugin';
+
+import * as TableTestUtils from '../module/test/TableTestUtils';
+
+describe('browser.tinymce.plugins.table.TableCellPropsStyleTest', () => {
+
+  const baseSettings = {
+    plugins: 'table',
+    base_url: '/project/tinymce/js/tinymce'
+  };
+
+  context('TINY-3909: remove_trailing_br is true', () => {
```

Comment:
We shouldn't need the Jira number here
```suggestion
  context('remove_trailing_br is true', () => {
```